### PR TITLE
Heading

### PR DESCRIPTION
**Pull request by TammyChen21**
fixed the problem
styled the homepage and each product card
US 1 is finished

### CODE REVIEW

File: components/Counter/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+
+//Counter component
+export default function Counter(props) {
+  return <StyledCounter>+</StyledCounter>;
```
Comment (mpagels) [inline components/Counter/index.js:6]: Is it really a counter? Looks like just a div with a `+` sign.
And you accepting the `props` object, but you are not using it. You can omit then the parameter.

File: components/ProductCard/index.js [hunk 2]
```diff
@@ -1,14 +1,63 @@
...
+        width="180px"
+        height="180px"
+      />
+      <div>
+        <StyledTitle>{props.productcard.title}</StyledTitle>
+        <StyledDes>{props.productcard.description}</StyledDes>
+        <StyledPrice>{props.productcard.price}</StyledPrice>
+      </div>
+      <div><Counter amount={props.productcard.amount} /></div>
+    </StyledProductCard>
```
Comment (mpagels) [inline components/ProductCard/index.js:20]: Might wanna use the fancy ES6 functionality `object desctructuring`

```suggestion
export default function ProductCard({productcard}) {
const {image, title, description, price, amount} = productcard
  return (
    <StyledProductCard>
      <StyledImg
        src={image}
        alt="photo"
        width="180px"
        height="180px"
      />
      <div>
        <StyledTitle>{title}</StyledTitle>
        <StyledDes>{description}</StyledDes>
        <StyledPrice>{price}</StyledPrice>
      </div>
      <div><Counter amount={amount} /></div>
    </StyledProductCard>
```

File: components/ProductGallery/index.js [hunk 3]
```diff
@@ -2,24 +2,19 @@ import React from "react";
...
-        <ProductCard/>
-        <ProductCard/>
-        <ProductCard/>
-        <ProductCard/>
+        {props.productData.map(item=>
+        <ProductCard 
+        key={item.id}
+        productcard={item}
+        /> 
+        )}
```
Comment (mpagels) [inline components/ProductGallery/index.js:14]: This looks strangely formatted. Is your prettier working correctly?

File: components/ProductGallery/index.js [hunk 4]
```diff
@@ -2,24 +2,19 @@ import React from "react";
 import styled from "styled-components";
 import ProductCard from "../ProductCard";
 
-export default function ProductGallery() {
+export default function ProductGallery(props) {
```
Comment (mpagels) [inline components/ProductGallery/index.js:5]: Please see my other comment about the ES6 functionality object desctructuring.

**Review by mpagels — COMMENTED** [review]
Hey Siqi, 
very good job here.

Just one issue with the `Counter` component, that is not looking like a counter.
And I found some minor improvements. Please have a look.

Good work, keep it up.

File: pages/index.js [hunk 5]
```diff
@@ -1,13 +1,113 @@
...
+  },
+
+  {
+    id: "12",
+    image: "../img/12.jpeg",
+    title: "Set Leggins Hoodie",
+    description: "süßes Set",
+    price: "10,00€",
+  },
+];
```
[the remark this answers is not in the document]
Comment (TammyChen21) [reply pages/index.js:102]: Thanks a lot for the feedbacks, i have fixed all problems.

**Review by stefankorder — APPROVED** [review]
LGTM!

**Pull request merged**